### PR TITLE
Replace @kiwicom/orbit-components with grommet

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: src/example-relay/src/Homepage/locations/Location.js [hunk 1]
```diff
@@ -20,12 +18,11 @@ function Location({ location }: Props) {
   const name = location.name ?? '';
   return (
     <li>
-      <Stack direction="row" spacing="compact">
-        <CountryFlag location={location} />
```
Comment (mrtnzlml) [inline src/example-relay/src/Homepage/locations/Location.js:24]: I believe there was something like `iconUrl` which you could use here. But no big deal.

File: src/example-relay/src/Homepage/locations/LocationsPaginated.js [hunk 2]
```diff
@@ -32,9 +32,7 @@ function LocationsPaginated(props: Props) {
           <Location key={edge?.node?.id} location={edge?.node} />
         ))}
       </ol>
-      <Button onClick={loadMore} size="small">
-        Load more!
-      </Button>
+      <Button label=" Load more!" onClick={loadMore} size="small" primary />
```
Comment (mrtnzlml) [inline src/example-relay/src/Homepage/locations/LocationsPaginated.js:35]:
```suggestion
      <Button label="Load more!" onClick={loadMore} size="small" primary />
```

File: src/example-relay/src/Homepage/locations/Location.js (hunk 1 above)
Comment (tbergq) [reply src/example-relay/src/Homepage/locations/Location.js:24]: Ahh, yes, we have a `countryFlagURL`, I see if I can re add it in a later PR

**Comment by tbergq** [comment]
> Nice one! 👍
> 
> How was your experience with `grommet`?

Yeah, it was good, I would definitely use it again 👍 

The most difficult was to find out how to extend the button styles 😂

**Pull request merged**